src/main/java/net/datafaker/transformations/sql/SqlTransformer.java

```diff
@@ -394,6 +394,63 @@ private String generateBatchModeStatements(Schema<IN, ?> schema, List<IN> inputs
         return sb.toString();
     }
 
+    /**
+     * Represents an interval of integers.
+     * Useful to capture range of rows corresponding to batch SQL statements.
+     *
+     * @see SqlTransformer#generateStream(Schema, long)
+     */
+    private static class Interval {
+        Integer start;
+        Integer end;
```

Comment:
```suggestion
        final Integer start;
        final Integer end;
```
Is there a reason to have it mot immutable?